pixi.toml

```diff
@@ -3,43 +3,58 @@
 [workspace]
 channels = ["conda-forge"]
 platforms = ["linux-64", "linux-aarch64", "win-64"]
+preview = ["pixi-build"]
 
-[pypi-dependencies]
-numba-cuda = { path = ".", editable = true }
+[workspace.build-variants]
+python = ["3.10.*", "3.11.*", "3.12.*", "3.13.*"]
```

Comment:
These can't quite be used yet because of https://github.com/prefix-dev/pixi/issues/4877.